db/migrations/20240529151056_Page.sql

```diff
@@ -0,0 +1,9 @@
+-- Ensure the UUID extension is enabled
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE Page (
```

Comment:
Can we call the table `Pages`? I think we'll do plural names for all our tables just as a convention